pom.xml

```diff
@@ -23,9 +23,9 @@
   </scm>
     <properties>
         <quarkus.version>3.18.2</quarkus.version>
-        <quarkus.angusmail.version>0.0.2</quarkus.angusmail.version>
-        <quarkus.poi.version>2.1.1</quarkus.poi.version>
+        <jakarta-mail-api.version>2.1.3</jakarta-mail-api.version>
```

Comment:
Wonder if we shouldn't also include the Jakarta mail API in the Quarkus BOM?

Reply:
if we include other Jakarta API's in the BOM that would make sense?

Reply:
Probably yes

Reply:
Actually we do have Jakarta mail already: https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L73

Just not the API, we should probably fix that

Reply:
should I open a Quarkus ticket and PR?

Reply:
PR opened: https://github.com/quarkusio/quarkus/pull/46164